processor/src/main/kotlin/permissions/dispatcher/processor/impl/kotlin/SystemAlertWindowHelper.kt

```diff
@@ -14,8 +14,8 @@ class SystemAlertWindowHelper : SensitivePermissionInterface {
         builder.beginControlFlow("if (%T.hasSelfPermissions(%N, *%N) || %T.canDrawOverlays(%N))", PERMISSION_UTILS, activity, permissionField, SETTINGS, activity)
     }
 
-    override fun addRequestPermissionsStatement(builder: FunSpec.Builder, activity: String, requestCodeField: String) {
-        builder.addStatement("val intent = %T(%T.ACTION_MANAGE_OVERLAY_PERMISSION, %T.parse(\"package:\" + %N.getPackageName()))", INTENT, SETTINGS, URI, activity)
-        builder.addStatement("%N.startActivityForResult(intent, %N)", activity, requestCodeField)
+    override fun addRequestPermissionsStatement(builder: FunSpec.Builder, targetParam: String, activityVar: String, requestCodeField: String) {
+        builder.addStatement("val intent = %T(%T.ACTION_MANAGE_OVERLAY_PERMISSION, %T.parse(\"package:\" + %N!!.getPackageName()))", INTENT, SETTINGS, URI, activityVar)
```

Comment:
We can replace `!!` with `?`. Like `parse("package: " + activity?.getPackageName())` it ends up `parse("package: null")`. What do you think ? @hotchemi @tomoya0x00 

However, if we could add null check to Java version, we can do early return in kotlin so that we don't need to worry about this problem.

```kotlin
fun ContactsFragment.fooWithPermissionCheck() {
  val activity = activity ?: return // add this 
  if (PermissionUtils.hasSelfPermissions(activity, *PERMISSION_FOO) || Settings.canDrawOverlays(activity)) {
    foo()
  } else {
    val intent = Intent(Settings.ACTION_MANAGE_OVERLAY_PERMISSION, Uri.parse("package:" + activity.getPackageName()))
    activity.startActivityForResult(intent, REQUEST_FOO)
  }
}
```

Reply:
I'm not sure we should do this in this PR. We can ship this PR and then later version we can add null check in both Java and Kotlin.

Reply:
I missed this comment.

I think we can add a null check later.

I think that `early return` is better. So We already use it in PermissionRequest class, I feel a sense of unity.

Reply:
Yea I definitely prefer the latter one. But it can be another PR for sure~